overview.md

```diff
@@ -0,0 +1,53 @@
+### Overview
+The flavour of the documentation is primarily educational. It contains papers,
+datasheets, style guidelines and team written documentation. The documentation
+is organized as follows:
+- `adcs`
+  - **Attitude Determination and Control System**: How
+the satellite can determine its orientation in space and reorient itself if
+necessary.
+  - *Topics*: Reaction wheels, IMU's, GPS, magnetorquers.
+- `comms`
+  - **Communication**: How the satellite communicates with the ground station.
+  - *Topics*: Transceivers, antenna design and simulation.
+- `general`
+  - **General team and space related documentation**: Contains things such
+as the competition rules and the critical design review.
+- `obc`
+  - **On Board Computer**: The main computer which organises and executes
+tasks on the satellite.
+  - *Topics:* Embedded system development, ARM processors, RTOS, STM32.
+- `power`
+  - **Electronic Power System**: How the satellite stores power from the solar
+  panels in batteries and distributes it to the other subsystems.
+  - *Topics:* Batteries, solar panels, PCB design.
+
```

Comment:
You could consider adding what discipline are applicable to each.  I think this would be beneficial. As an example, as a new member I would not have knows there was a lot of math involved with the ADCS.  For someone in apple or considering it this would be good to know.